### DISCUSSION TITLE
Update process no update found

### DISCUSSION THREAD

**Discussion by caywin25** [FreshRSS/FreshRSS#6500] [Q&A]
Hello

I could see that version 1.24 was released.
Thanks :)
When I'm on my freshrss instance and I go to options and Update, if I click: check for updates, it doesn't find any updates.

Is it because I installed freshrss from a zip archive that it can't find an update?

If so, how can I resolve the issue please?
Thanks

**Comment by Frenzie** [comment]
See https://github.com/FreshRSS/FreshRSS/issues/6494#issuecomment-2131343493

**Reply by caywin25** [reply]
Oh ok thank you :)

I will wait then
Regards
